src/plugins/discover/public/application/components/discover.tsx

```diff
@@ -26,18 +26,25 @@ import classNames from 'classnames';
 import { HitsCounter } from './hits_counter';
 import { TimechartHeader } from './timechart_header';
 import { getServices } from '../../kibana_services';
-import { DiscoverUninitialized, DiscoverHistogram } from '../angular/directives';
+import { DiscoverHistogram, DiscoverUninitialized } from '../angular/directives';
 import { DiscoverNoResults } from './no_results';
 import { LoadingSpinner } from './loading_spinner/loading_spinner';
+import { DocTableLegacy, DocTableLegacyProps } from '../angular/doc_table/create_doc_table_react';
+import { SkipBottomButton } from './skip_bottom_button';
 import { search } from '../../../../data/public';
 import {
   DiscoverSidebarResponsive,
   DiscoverSidebarResponsiveProps,
 } from './sidebar/discover_sidebar_responsive';
-import { DiscoverProps } from './discover_legacy';
+import { DiscoverProps } from './types';
+import { getDisplayedColumns } from '../helpers/columns';
 import { SortPairArr } from '../angular/doc_table/lib/get_sort';
 import { DiscoverGrid, DiscoverGridProps } from './discover_grid/discover_grid';
+import { SEARCH_FIELDS_FROM_SOURCE } from '../../../common';
 
+export const DocTableLegacyMemoized = React.memo((props: DocTableLegacyProps) => (
```

Comment:
nit: do we need to export this?

Reply:
definitely not! like 3 other exports here, good catch, removed them